Ballerina-Training/Ballerina-Forum-Application/back-end/bal-forum/types.bal

```diff
@@ -0,0 +1,193 @@
+import ballerina/http;
+import ballerina/time;
+
+# Represents the new user request
+type NewUser record {|
```

Comment:
Is it possible to have better naming for these records?
`NewUser`, `User`, `UserDB` - These are not self-explanatory and bit confusing?
Same goes with `Post` types

Reply:
Will give better names

Reply:
Fixed via [1eae7ab](https://github.com/ballerina-guides/training-content/pull/12/commits/1eae7abbc8d1d6bdd3a677b4fac348875ee53fb7)